Tasks/DotNetCoreCLIV2/task.json

```diff
@@ -262,7 +262,7 @@
             "name": "verbosityRestore",
             "type": "pickList",
             "label": "Verbosity",
-            "defaultValue": "Detailed",
+            "defaultValue": "Minimal",
```

Comment:
It was previously decided that the default value should be `Normal`. See https://github.com/microsoft/azure-pipelines-tasks/issues/12297#issuecomment-582506056